components/Footer/src/index.scss

```diff
@@ -0,0 +1,95 @@
+.denhaag-footer {
+  --utrecht-paragraph-color: #fff;
```

Comment:
Hmmm I'm thinking maybe these should come from a token too?

`--utrecht-paragraph-color: var(--denhaag-footer-color)`